floyd/cli/auth.py

```diff
@@ -12,13 +12,19 @@
 
 @click.command()
 @click.option('--token', is_flag=True, default=False, help='Just enter token')
+@click.option('--apikey', '-k', default=None, help='Api Key')
 @click.option('--username', '-u', default=None, help='FloydHub username')
 @click.option('--password', '-p', default=None, help='FloydHub password')
-def login(token, username, password):
+def login(token, apikey, username, password):
     """
     Log into Floyd via Auth0.
     """
-    if token:
+    if apikey and username:
+        AuthConfigManager.set_apikey(username=username, apikey=apikey)
```

Comment:
what happens if the user puts in the wrong username (a team) or made a typo?

Reply:
the actual auth is still checked by backend, it'll respond with 401 error